js/src/components/app.js

```diff
@@ -62,14 +62,16 @@ export default class App extends Component {
   }
   nextPageHandler = (e) => {
     e.preventDefault();
-    this.fetchLogEntries(this.state.page + 1);
+    this.state.page++;
```

Comment:
Do you mind using [this.setState](https://reactjs.org/docs/state-and-lifecycle.html) here instead?

Reply:
Yeah, I just caught this myself.